low/low.h

```diff
@@ -615,3 +615,15 @@ static int KNI_config_promiscusity(uint16_t port_id, uint8_t to_on) {
 	fprintf(stderr, "DEBUG: KNI: Promiscusity mode of port %d %s\n", port_id, to_on ? "on" : "off");
 	return 0;
 }
+
+bool check_hwtxchecksum_capability(uint16_t port_id) {
```

Comment:
You should return false also if port index exceeded existing ports number

Reply:
Ok